src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/Utils/XBindExpressionInfo.cs

```diff
@@ -125,7 +136,38 @@ public override string ToString()
 			}
 
 			builder.AppendLine("}");
-			return builder.ToString();
+			return pool.ToStringAndFree();
+		}
+
+		private string ToStringLValue()
+		{
+			var pool = PooledStringBuilder.GetInstance();
+			var builder = pool.Builder;
+			builder.AppendLine($"private bool TrySetInstance_xBind_{_xBindCounter}({_contextType} {_contextName}, object __value)");
+			builder.AppendLine("{");
```

Comment:
`BlockInvariant` is probably better to avoid opening a closing braces.

Reply:
We will need one of two options to do this:

- Introduce BlockInvariant extension method on `StringBuilder` itself.
- Introduce pooling for `IndentedStringBuilder`s with the ability to also pool the inner `StringBuilder`.

If we'll want to do this, I'd prefer the first option for simplicity. But it's a small block of code anyways.

Reply:
Ah I see, it's not the same builder. Let's skip for now.